quickcheck-threatmodel/src/Test/QuickCheck/ThreatModel/Cardano/Api.hs

```diff
@@ -229,6 +231,9 @@ validateTx pparams tx utxos = case result of
     systemStart :: SystemStart
     systemStart = SystemStart $ posixSecondsToUTCTime 0
 
+    genesisWindow :: GenesisWindow
+    genesisWindow = GenesisWindow 10
```

Comment:
Why is `10` a reasonable choice here?

Reply:
Not sure. There were several default fixtures in various places. Here is one of them.

https://github.com/IntersectMBO/ouroboros-consensus/blob/main/ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/History/EraParams.hs#L159